### PR TITLE
Feature mma 465

### PR DESCRIPTION
**Pull request by holiday-sunrise**

### CODE REVIEW

File: mule-migration-tool-api/src/main/java/com/mulesoft/tools/migration/project/model/pom/Parent.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
+/*
```
Comment (mlischetti) [inline mule-migration-tool-api/src/main/java/com/mulesoft/tools/migration/project/model/pom/Parent.java:1]: This change is not need to fix the issue, please remove it

File: mule-migration-tool-api/src/main/java/com/mulesoft/tools/migration/project/model/pom/PomModel.java [hunk 2]
```diff
@@ -156,6 +156,24 @@ public void setArtifactId(String artifactId) {
     model.setArtifactId(artifactId);
   }
 
+  /**
```
Comment (mlischetti) [inline mule-migration-tool-api/src/main/java/com/mulesoft/tools/migration/project/model/pom/PomModel.java:159]: This changes are not need to fix the issue, please remove it

**Review by mlischetti — CHANGES_REQUESTED** [review]
Hi @holiday-sunrise thanks for contributing to mma project.
Can you please isolate the core changes involved in this issue to make it mergeable?

**Comment by holiday-sunrise** [comment]
how should i  isolate he core changes

**Comment by holiday-sunrise** [comment]
Files have been removed

**Comment by mlischetti** [comment]
@holiday-sunrise, there is a misunderstood about PomModel.java file.
What I was trying to said is that changes in PomModel.java are not needed not that you should remove the file itself.
Also an unit test would be great to add on current PR

**Comment by mlischetti** [comment]
PomModel.java file still contains unnecessary modification please revert this changes

**Comment by holiday-sunrise** [comment]
Pom model should now reverted. sorry my mistake

**Review by mlischetti — APPROVED** [review]
LGTM

**Pull request merged**